src/pkg/config/manager.go

```diff
@@ -188,7 +190,27 @@ func (c *CfgManager) ValidateCfg(ctx context.Context, cfgs map[string]interface{
 		if item.Scope == metadata.SystemScope {
 			return fmt.Errorf("system config items cannot be updated, item: %v", key)
 		}
+
 		strVal := utils.GetStrValueOfAnyType(value)
+
+		// check storage per project before setting it
+		if key == common.StoragePerProject {
```

Comment:
why put the specific checking at here?

Reply:
Because the check is only for `common.StoragePerProject`